tools/gen_ucd_tables.py

```diff
@@ -28,12 +28,12 @@
 from common import path, memoize
 from unicode_utils import is_surrogate
 
-
```

Comment:
nit: Python's PEP-8 recommends two empty lines here, so would be great to keep it.

You can just run the `.cargo/format.sh` script to get it fixed if you have the CLI tool installed.